src/main/java/com/example/TwoRoomsBoom/models/Lobby.java

```diff
@@ -0,0 +1,25 @@
+package com.example.TwoRoomsBoom.models;
+
+import java.util.Set;
+import java.util.List;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.Builder.Default;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Lobby {
+    private String lobbyId;
+    private List<Role> rolesAdded;
+    private List<Player> players;
+    private Set<Player> readyPlayers;
+    private Player host;
+    private String settings;
```

Comment:
Just thinking about this now, how should we setup settings? Should we make an array containing a boolean for bury cards and ints for amount of rounds and time?

Reply:
Probably I think that maybe a Settings class would be helpful potentially?

Reply:
Like as I was making it I was wondering if there should be a dedicated class to hold the data values of the settings for that lobby

Reply:
You're right, a Settings class would probably be ideal, in the case that we have more things to add later down the line